Source/WebKit/WebKit.xcodeproj/project.pbxproj

```diff
@@ -2469,6 +2469,8 @@
 		F454C12A2BA3ABDC00871551 /* _WKTargetedElementInfo.h in Headers */ = {isa = PBXBuildFile; fileRef = F454C1272BA3ABDC00871551 /* _WKTargetedElementInfo.h */; settings = {ATTRIBUTES = (Private, ); }; };
 		F454C12C2BA3AC8C00871551 /* _WKTargetedElementInfoInternal.h in Headers */ = {isa = PBXBuildFile; fileRef = F454C12B2BA3AC8C00871551 /* _WKTargetedElementInfoInternal.h */; };
 		F454C1302BA3AFFD00871551 /* APITargetedElementInfo.h in Headers */ = {isa = PBXBuildFile; fileRef = F454C12D2BA3AFFD00871551 /* APITargetedElementInfo.h */; };
+		F458B8EA2E8F46E900948328 /* CoreIPCPKDateComponentsRange.h in Headers */ = {isa = PBXBuildFile; fileRef = F458B8E72E8F46E100948328 /* CoreIPCPKDateComponentsRange.h */; };
+		F458B8EB2E8F46F000948328 /* CoreIPCPKDateComponentsRange.mm in Sources */ = {isa = PBXBuildFile; fileRef = F458B8E82E8F46E100948328 /* CoreIPCPKDateComponentsRange.mm */; };
```

Comment:
Should you be adding this as input to the unified build concatenation process instead? I'm not sure of the guidelines, or whether this necessarily results in a standalone build step here